src/frontend/src/optimizer/plan_node/stream_source.rs

```diff
@@ -55,9 +55,10 @@ impl fmt::Display for StreamSource {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(
             f,
-            "StreamSource {{ source: {},  columns: [{}] }}",
+            "StreamSource {{ source: {},  columns: [{}], appendonly: {} }}",
```

Comment:
only show `append only: ` when it is set to true, so as to affect as little test cases as possible

Reply:
I grep the tests in test_runner, there is no test rely on this string. Seems fine to only affect new tests.